dspace-server-webapp/src/test/java/org/dspace/app/rest/CommunityAdminGroupRestControllerIT.java

```diff
@@ -437,4 +457,376 @@ public void deleteCommunityAdminGroupNotFoundTest() throws Exception {
         getClient(token).perform(delete("/api/core/communities/" + UUID.randomUUID() + "/adminGroup"))
                         .andExpect(status().isNotFound());
     }
+
+    @Test
+    public void communityAdminAddMembersToCommunityAdminGroupPropertySetToFalse() throws Exception {
+
+        context.turnOffAuthorisationSystem();
+        Group adminGroup = communityService.createAdministrators(context, parentCommunity);
+        authorizeService.addPolicy(context, parentCommunity, Constants.ADMIN, eperson);
+        EPerson ePerson = EPersonBuilder.createEPerson(context).withEmail("testToAdd@test.com").build();
+        configurationService.setProperty("core.authorization.community-admin.admin-group", false);
+        context.restoreAuthSystemState();
+
+
+        String token = getAuthToken(eperson.getEmail(), password);
+        getClient(token).perform(
+            post("/api/eperson/groups/" + adminGroup.getID() + "/epersons")
+                .contentType(parseMediaType
+                    (org.springframework.data.rest.webmvc.RestMediaTypes.TEXT_URI_LIST_VALUE))
+                .content("https://localhost:8080/server/api/eperson/epersons/" + ePerson.getID()))
+                        .andExpect(status().isForbidden());
+
+        token = getAuthToken(admin.getEmail(), password);
+        getClient(token).perform(get("/api/eperson/groups/" + adminGroup.getID() + "/epersons"))
+                        .andExpect(status().isOk())
+                        .andExpect(jsonPath("$._embedded.epersons", Matchers.not(Matchers.hasItem(
+                            EPersonMatcher.matchEPersonOnEmail(ePerson.getEmail())
+                        ))));
+
+        context.turnOffAuthorisationSystem();
+        configurationService.setProperty("core.authorization.community-admin.admin-group", true);
+        context.restoreAuthSystemState();
```

Comment:
You don't need these two context lines here. You can change a configuration property in tests without turning off the authorization system.  Also, the configuration system resets itself after every test, so you may be able to remove all three lines here...you need not reset it back to the default value.